server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java

```diff
@@ -31,6 +38,7 @@
 public class IndexerStartupTask {
 
   private static final Logger LOG = Loggers.get(IndexerStartupTask.class);
+  private static final String SETTING_PREFIX_INITIAL_INDEXING_FINISHED = "initial_indexing_finished.";
```

Comment:
It's not common to put its own settings in Elasticsearch. In order to prevent any misundertanding I recommend to prefix name by "sonarqube_".